src/models/Store.js

```diff
@@ -0,0 +1,21 @@
+import { observable } from "mobx";
+
+import User from './User';
+
+export const checked = 'checked';
+export const empty = 'empty';
+export const loading = 'loading';
+
+class Store {
+  @observable status = 'empty';
```

Comment:
Here we make the variables "observable" which allows the observer to keep track of them.